srcpkgs/libpdal/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'libpdal'
+pkgname=libpdal
+version=2.5.6
+revision=1
+archs="x86_64* i686"
```

Comment:
why only x86?

Reply:
I didn't get it to compile on arm because it uses some vendored tool `dimbuilder` on which it failed with `/bin/sh: line 1: dimbuilder: command not found`. It's also not entirely clear on which architectures it's [supposed to compile](https://github.com/PDAL/PDAL/issues/4239), though probably it should work on some arm arches. Any tips what to try there? Might `nocross` help here?

Reply:
i think nocross is probably reasonable